packages/core/src/components/essentials/title.ts

```diff
@@ -1,12 +1,17 @@
 // Internal Imports
 import { Component } from "../component";
 import { DOMUtils } from "../../services";
-import { Events } from "./../../interfaces";
+import { AxisPositions, Events, ScaleTypes } from "./../../interfaces";
+import { Tools } from "../../tools";
+import * as Configuration from "../../configuration";
 
 export class Title extends Component {
 	type = "title";
 
 	render() {
+		const toolbarEnabled =
```

Comment:
title shouldn't need to know about toolbar

Reply:
Fixed in https://github.com/carbon-design-system/carbon-charts/pull/717/commits/a734ef0502eede584e05dc56519e96f2f9a92a00.